libse/SpellCheck/SpellCheckWordLists.cs

```diff
@@ -81,10 +79,12 @@ public SpellCheckWordLists(string dictionaryFolder, string languageName, IDoSpel
                     }
                 }
             }
-
             // Add names/userdic with "." or " " or "-"
-            _wordsWithDashesOrPeriods = new List<string>();
-            _wordsWithDashesOrPeriods.AddRange(namesEtcMultiWordList);

```

Comment:
Here we are just copying all the element s of namesEtcMultiWordList to _wordsWithDashesOrPeriods which is not correct, we need to make sure that the element being copied as **period** or **dash**.
See https://github.com/SubtitleEdit/subtitleedit/pull/2281/files#diff-9e4e72215c5f592ba3b97997bfed63feR95